### PR TITLE
🔧 ProfileLink, ScrollTopButton, HotTable & SalesTable

### LINKED ISSUE #6141: back to top does weird overlay apparently
On mobile, back to top does weird overlay apparenlty.

image

image

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/6118#issuecomment-1569974098_

### LINKED ISSUE #6609: Hot page shows different results, but they are all RMRK1
### What happened?

Go to Kusama Chain -> access /hot from browser bar, check results
change to RMRK2 -> access /hot, check results
image
image

### PR DESCRIPTION
**Pull request by roiLeo**
- part of #4750 
- close #6141 
- close #6609 
- `/hot` page composition api
Screenshot 2023-08-07 at 11-45-49 Recently KUSAMA Sales KodaDot - NFT Market Explorer
- `/sales` page composition api
Screenshot 2023-08-07 at 19-32-10 Latest KUSAMA Sales KodaDot - NFT Market Explorer

edit:
- updated page subtitle to display current chain
Screenshot 2023-08-07 at 19-49-36 Recently KUSAMA Sales KodaDot - NFT Market Explorer

### CODE REVIEW

**Comment by vikiival** [comment]
maybe different logo without background?

Screenshot 2023-08-07 at 13 14 37


[Kusama Branding Assets](https://kusama.network/assets/kusama-brand-assets.zip)

File: styles/components/_scroll-to-top-button.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .scroll-top-button {
   position: fixed !important;
-  right: 24px;
+  right: 42px;
```
Comment (vikiival) [inline styles/components/_scroll-to-top-button.scss:3]: Typo?

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
Opened issue for the data view
- https://github.com/kodadot/loligo/issues/43

File: styles/components/_scroll-to-top-button.scss (hunk 1 above)
Comment (roiLeo) [reply styles/components/_scroll-to-top-button.scss:3]: > Typo?

close #6141 to please mr @yangwao

**Review by daiagi — COMMENTED** [review]
looks good, couple of q please
1. how to navigate to this page (hot)? (i.e do we even need it anymore?)
2. could you move it under prefix? 

image

**Comment by roiLeo** [comment]
> looks good, couple of q please
> 
>     1. how to navigate to this page (hot)? (i.e do we even need it anymore?)
> 
>     2. could you move it under prefix?

If I remember we hide it a while ago, it will be reintroduce once we'll fixed all stuff in StatsDropdown (series, spotlight, sales, hot etc..)

**Comment by prury** [comment]
regarding showing identity on the profile page, even if a user has it in its identity, we are not currently showing social media handles, right?

**Comment by roiLeo** [comment]
> regarding showing identity on the profile page, even if a user has it in its identity, we are not currently showing social media handles, right?

I think we are showing them in profile detail page (`/:prefix/u/:id`)

**Pull request merged**